lib/ts/recipe/session/index.ts

```diff
@@ -469,7 +509,7 @@ export let removeClaim = SessionWrapper.removeClaim;
 export let validateClaimsInJWTPayload = SessionWrapper.validateClaimsInJWTPayload;
 export let validateClaimsForSessionHandle = SessionWrapper.validateClaimsForSessionHandle;
 
-export let Error = SessionWrapper.Error;
+export let STError = SessionWrapper.STError;
```

Comment:
This makes it a breaking change, is that intentional?

Reply:
@rishabhpoddar